### DISCUSSION TITLE
New hardware: notepad HP Compaq 6910p

### DISCUSSION THREAD

**Discussion by defdefred** [VendelinSlezak/BleskOS#33] [Show and tell]
Wav: ok but no sound
Touchpad: ok
keyboard: ok
ethernet: not tested.
VBE: ok 1024x768 but not max resolution of the screen 1280x800, so blurry. Could be better to set   1280x1024 with an non displayed zone?
[hpcompaq6910pu14.txt](https://github.com/VendelinSlezak/BleskOS/files/13760406/hpcompaq6910pu14.txt)

**Comment by VendelinSlezak** [comment]
> VBE: ok 1024x768 but not max resolution of the screen 1280x800, so blurry. Could be better to set 1280x1024 with an non displayed zone?

You can try it and see how does it look like. In bootloader go to [a] Boot options>[g] Select graphic mode>Use arrows up and down to select mode, then go back by ESC and select "BleskOS special boot options". https://github.com/VendelinSlezak/BleskOS/wiki/Booting-of-BleskOS

**Comment by defdefred** [comment]
Tried with version 15, but I don't have better option than 1024x768x32

Internet access if working and I must say that wiki.osdev.org page is impressive in the internet browser. Great job!
